src/v2/BaseLoginRouter.js

```diff
@@ -80,6 +80,7 @@ export default Router.extend({
     configIdxJsClient(this.appState);
     this.listenTo(this.appState, 'remediationSuccess', this.handleIdxResponseSuccess);
     this.listenTo(this.appState, 'remediationError', this.handleIdxResponseFailure);
+    this.listenTo(this.appState, 'interactionCanceled', this.startLoginFlow);
```

Comment:
does it not work if you do `this.render` instead of `this.startLoginFlow`

Reply:
No, this would produce error `TypeError: Controller is not a constructor` in this line:
https://github.com/okta/okta-signin-widget/blob/8d27ecb7f46fb1ea11ad49cf8538b661de546239/src/v2/BaseLoginRouter.js#L247

Reply:
Changed `render` method for using it for re-render current controller (without parameters)